chart/values.yaml

```diff
@@ -756,3 +756,7 @@ localpv-provisioner:
     enabled: false
   hostpathClass:
     enabled: false
+  # Create a hostpath storage class for mayastor-etcd.
+  createEtcdSc: true
+  # Create a hostpath storage class for Loki Stack.
+  createLokiSc: true
```

Comment:
These values are used by the mayastor chart and do not need to be passed down to the localpv-provisioner helm dependency.

Reply:
Could we maybe use something along the lines of `etcd.localpvScConfig.enabled: true` and `loki-stack.localpvScConfig.enabled: true`?

Reply:
It's related to localpv-provisoner version of the mayastor chart so it probably makes sense to keep it here?

Reply:
It's not related to the localpv chart at all, I'd guess. StorageClasses are used by loki and etcd, localpv doesn't need the resources this enables/disables.

Reply:
Okay. I will move it there. @tiagolobocastro @avishnu Does this sound good?

Reply:
Sure, sounds good to me, please update PR description as appropriate